src/core/public/chrome/ui/header/header_breadcrumbs.scss

```diff
@@ -0,0 +1,49 @@
+@import "../../../../public/variables";
+
+@mixin backround($color,$shape) {
+  background-color: #{$color};
+  clip-path: $shape;
+  padding: $euiSizeXS - 2.5 $euiSizeL - $euiSizeXS;
+  border-radius: $euiSizeXS - 2;
+}
+
+.osdHeaderBreadcrumbs {
+  [data-background="osdHeaderBreadcrumbs first"] {
+    @include backround($osdSanFranciscoFog, polygon(0% 0%,100% 0%,calc(100% - #{
+      $euiSizeS + 2}) 100%,0% 100%));
```

Comment:
- Why are we using `$euiSizeS + 2` which is 6px? can it not be either 4 or 8? The values supported by eui?
- We should be explicit about the units here. e.g. `2px`

Reply:
here, we are trying to mock ux designs! $euiSizeS + 2 represents 10px! 4/8 will be too smaller

Reply:
we have 12px too ;)

Reply:
I am going with 8! 12 might be little bigger